pufferlib/ocean/torch.py

```diff
@@ -172,3 +172,118 @@ def decode_actions(self, flat_hidden, lookup, concat=None):
             #print('argmax samples: ', argmax_samples)
 
             return action, value
+
+
+class TrashPickup(nn.Module):
```

Comment:
This architecture is nuts
1. Combine all these small linear nets so you don't end up launching a ton of tiny cuda kernels. You can just concat features
2. Don't use hidden size 8. 16 is sketch, 32 should be bare minimum. You lose properties of high-dim optimization when you go this small, and everything will just feel cursed
3. Net does not need to be this deep. conv-conv-linear-lstm-atn_proj for 2d or linear-linear-lstm-atn_proj for 1d is a better starting point. Deep nets do not work well in RL, esp. without careful skip connects etc.

Reply:
I would suggest:
- concat everything
- linear from that to 128, relu, 128, relu
- 128 lstm
- atn proj

Reply:
Fixed.